### DISCUSSION TITLE
Add mutation testing

### DISCUSSION THREAD

**Discussion by wilgert** [dinerojs/dinero.js#240] [Ideas]
The aim of mutation testing is to improve the quality of the unit tests, discover untested code paths and possibly find bugs in both tests and production code.

Full introduction here: https://stryker-mutator.io/docs/

**Comment by sarahdayan** [comment]
Hey @wilgert, and thanks for the suggestion.

**What problem would this solve? Have you noticed something problematic in the codebase that this tool would avoid?**

New dependencies mean more maintenance and potentially a steeper learning curve for new contributors, so I don't take introducing new tools lightly. I'm open to chat about this though!
